## Distribución binomial

En 460 partidos de la NFL, 252 se han ganado por el equipo que gana el volado inicial. ¿Es puro azar o representa una ventaja significativa?

Hallar la probabilida de 252 victorias o más en 460 partidos, cada uno con una probabilidad de 0.5

In [22]:
p252omas <- sum(dbinom(x=252:460, size=460, prob=0.5))

In [23]:
p252omas

[1] 0.02242894

In [24]:
sum (p252omas)

[1] 0.02242894

# rango de significancia


$$ \mu =268 $$
$$ \sigma = 15 $$
$$ rango = \mu \pm 2\sigma $$

In [28]:
mu <- 460*0.5
ancho <- 2*sqrt(460*0.5*0.5)

In [29]:
mu + ancho

[1] 251.4476

In [30]:
mu - ancho

[1] 208.5524

Por lo que el rango de significancia se encuentra entre 208.55 y 251.44

Al salirse los 252 partidos del rango, significa que nos aleatorio que se ganen tantos partidos